operator/hack/lokistack_gateway_ocp.yaml

```diff
@@ -3,11 +3,19 @@ kind: LokiStack
 metadata:
   name: lokistack-dev
 spec:
-  size: 1x.extra-small
+  size: 1x.small
   storage:
     secret:
       name: test
       type: s3
   storageClassName: gp2
   tenants:
     mode: openshift-logging
+  rules:
+    enabled: true
```

Comment:
Ruler enabled for all our testing clusters in the future? Or should we have a separate config file?

Reply:
I opt for using the ruler in testing by default in the `_ocp` suffixed hack manifests as per ruler will be an important aspect for user buy-in.